### DISCUSSION TITLE
can i use this engine with the web version?

### DISCUSSION THREAD

**Discussion by TheRealFowluhh** [MrMeowzz/Funkin-MrMeowzz#59] [Q&A]
i am not able to run haxe on my computer...

**Comment by MrMeowzz** [comment]
https://mrmeowzz.github.io/Funkin-MrMeowzz/web/

**Reply by TheRealFowluhh** [reply]
i actually meant i wanted to mod the web version, am i able to?
if i am then thanks

(sorry for being talkative...)

**Reply by MrMeowzz** [reply]
well you need haxe to be able to compile it if you want to mod it
unless you just want to change the assets then all you need to do is upload the html files with the changed files on your own repo with github pages

**Reply by TheRealFowluhh** [reply]
yea, that is basically what i wanna do

thank you

**Reply by TheRealFowluhh** [reply]
i was wanting permissions thats all or smthn idk

**Reply by MrMeowzz** [reply]
yea you can i really dont care
